MonoGame.Framework/Input/GamePadCapabilities.cs

```diff
@@ -1,100 +1,180 @@
-a complete copy of this license with your distribution. If you distribute any portion of the software in compiled or object 
-code form, you may only do so under a license that complies with this license.
-(E) The software is licensed "as-is." You bear the risk of using it. The contributors give no express warranties, guarantees
-or conditions. You may have additional consumer rights under your local laws which this license cannot change. To the extent
-permitted under your local laws, the contributors exclude the implied warranties of merchantability, fitness for a particular
-purpose and non-infringement.
-*/
-#endregion License
-
-using System;
-
-namespace Microsoft.Xna.Framework.Input
-{
-    public struct GamePadCapabilities
-    {
-        public bool IsConnected { get; internal set; }
-
-		public bool HasAButton { get; internal set; }
-
-		public bool HasBackButton { get; internal set; }
-
-		public bool HasBButton { get; internal set; }
-
-		public bool HasDPadDownButton { get; internal set; }
-
-		public bool HasDPadLeftButton { get; internal set; }
-
-		public bool HasDPadRightButton { get; internal set; }
-
-		public bool HasDPadUpButton { get; internal set; }
-
-		public bool HasLeftShoulderButton { get; internal set; }
-
-		public bool HasLeftStickButton { get; internal set; }
-
-		public bool HasRightShoulderButton { get; internal set; }
-
-		public bool HasRightStickButton { get; internal set; }
-
-		public bool HasStartButton { get; internal set; }
-
-		public bool HasXButton { get; internal set; }
-
-		public bool HasYButton { get; internal set; }
-
-		public bool HasBigButton { get; internal set; }
-
-		public bool HasLeftXThumbStick { get; internal set; }
-
-		public bool HasLeftYThumbStick { get; internal set; }
-
-		public bool HasRightXThumbStick { get; internal set; }
-
-		public bool HasRightYThumbStick { get; internal set; }
-
-		public bool HasLeftTrigger { get; internal set; }
-
-		public bool HasRightTrigger { get; internal set; }
-
-		public bool HasLeftVibrationMotor { get; internal set; }
-
-		public bool HasRightVibrationMotor { get; internal set; }
-
-		public bool HasVoiceSupport { get; internal set; }
-
-        public GamePadType GamePadType { get; internal set; }
-    }
-}
-
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+
+namespace Microsoft.Xna.Framework.Input
+{
+    public struct GamePadCapabilities
+    {
+        /// <summary>
+        /// Gets a value indicating whether the gamepad is connected.
+        /// </summary>
+        /// <value><c>true</c> if the gamepad is connected; otherwise, <c>false</c>.</value>
+        public bool IsConnected { get; internal set; }
+
+        /// <summary>
+        /// Gets the gamepad display name.
+        /// </summary>
+        /// <value>String representing the display name of the gamepad..</value>
```

Comment:
> IMO that piece of information isn't important enough to be in summary.

I don't love having it in the summary...  but it is the only place to put it that makes it visible to you as you write code.

> Also we have several other APIs that weren't in XNA, and we are not marking them in any way.

I am fully aware... and I bring this up because I think it is a problem.  As someone that has ported around a dozen XNA/MG titles to consoles... most of which use XNA on Windows desktop releases... knowing if you are coding to an API that isn't cross platform with XNA is useful.